syntax = "proto3";

service MyService {
  // Existing method
  rpc GetMessage (MessageRequest) returns (MessageResponse);

  // New method for creating a user
  rpc CreateUser (CreateUserRequest) returns (CreateUserResponse);
}

message MessageRequest {
  string name = 1;
}

message MessageResponse {
  string message = 1;
}

message CreateUserRequest {
  string username = 1;
  string email = 2;
  string password = 3;
}

message CreateUserResponse {
  bool success = 1;
  string message = 2;
}